src/test/java/com/puppycrawl/tools/checkstyle/PackageObjectFactoryTest.java

```diff
@@ -397,6 +416,11 @@ public void testGetShortFromFullModuleNamesThirdParty() {
                 PackageObjectFactory.getShortFromFullModuleNames(fullName));
     }
 
+    /**
+     * Test check for {@link PackageObjectFactoryTest#testConstructorFailure()}.
+     *
+     * @noinspection JUnitTestCaseWithNoTests
```

Comment:
We try to keep reason of suppression each time close to suppression.

Please explain reason of suppression in javadoc description.

Reply:
done

Reply:
done